### PR TITLE
Added buttons and FIX bugs 

### PR DESCRIPTION
**Pull request by jorisadri**
Added buttons to:

-accout
-publicAccount
-listener
-accountInfo

FIX for:
 -mosaicSupplyChange, wrong mosaic added
-signTransaction cosignature convert to correct transaction
-aggregateBondendTransaction did loose account information

### CODE REVIEW

File: NEMNodes/account/account.html [hunk 1]
```diff
@@ -29,6 +29,40 @@
...
+        },
+        button: {
+            enabled: function() {
+                return !this.changed
+            },
+            onclick: function() {
+                if (this.changed) {
+                    return RED.notify(RED._("notification.warning", {message:RED._("notification.warnings.undeployedChanges")}),"warning");
+                }
+                if(this.privateKey =="")
```
Comment (aleixmorgadas) [inline NEMNodes/account/account.html]: `!this.privateKey.trim()`

To check it's not blank or empty. Should we check here the size of the private key and if it's valid?

File: NEMNodes/account/account.html [hunk 2]
```diff
@@ -29,6 +29,40 @@
...
+                }
+                var label = (this.name||"account");
+                var node = this;
+                $.ajax({
+                    url: "inject/"+this.id,
+                    type:"POST",
+                    success: function(resp, textStatus, xhr) {
+                        RED.notify(node._("succesfully injected "+label),"success");
+                    },
+                    error: function(jqXHR,textStatus,errorThrown) {
```
Comment (aleixmorgadas) [inline NEMNodes/account/account.html]: I guess this function is common across different `nodes` & `buttons`. Refactor

Comment (aleixmorgadas) [reply NEMNodes/account/account.html]: In fact, this commit has this code 4 times already

File: NEMNodes/transactions/aggregateBondedTransaction.js [hunk 3]
```diff
@@ -30,10 +31,13 @@ module.exports = function (RED) {
                 if (typeof msg.nem === "undefined") {
                     msg.nem = {};
                 }
-                const publicKey = msg.nem.publicKey;//get publicKey from account(public and full account)
-
+                //get publicKey from account(public and full account)
+                const publicKey = msg.nem.publicKey;
                 // save the transactions until the trigger arives
                 let savedTransactions = context.get(node.id) || [];
+                if (account === "" && msg.nem.account != undefined) {
```
Comment (aleixmorgadas) [inline NEMNodes/transactions/aggregateBondedTransaction.js]: I'm not sure how safe is this operation, could you add tests to be sure of edge cases?

something I would test is:
- account = ""
- account = " "
- account = "asdas 2384"

and so on

File: NEMNodes/account/account.html (hunk 2 above)
Comment (jorisadri) [reply NEMNodes/account/account.html]: In node-red its hard to have global functions in the script in the html files.
